tests/lib/config.js

```diff
@@ -1055,6 +1055,205 @@ describe("Config", () => {
                 }, "No ESLint configuration found");
             });
         });
+
+
+        describe("with overrides", () => {
+
+            /**
+             * Returns the path inside of the fixture directory.
+             * @returns {string} The path inside the fixture directory.
```

Comment:
Fixed (though there are a bunch of other functions in this file that use the same pattern that could probably use a refactor).